packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettings.tsx

```diff
@@ -28,11 +28,19 @@ export const DatabaseSettings: React.FC<Props> = (props) => {
     </Typography>
   );
 
-  const resetRootPasswordCopy =
-    'Resetting your root password will automatically generate a new password. You can view the updated password on your database cluster summary page. ';
+  let resetRootPasswordCopy =
+    'Reset your root password if someone should no longer have access to the root user or if you believe your password may have been compromised. This will automatically generate a new password that you’ll be able to see on your database cluster summary page.';
 
-  const deleteClusterCopy =
-    'Deleting a database cluster is permanent and cannot be undone.';
+  let deleteClusterCopy = 'Permanently remove an unused database cluster.';
+
+  const isLegacy = database.platform === 'rdbms-legacy';
+  if (isLegacy) {
+    resetRootPasswordCopy =
+      'Resetting your root password will automatically generate a new password. You can view the updated password on your database cluster summary page. ';
+
+    deleteClusterCopy =
+      'Deleting a database cluster is permanent and cannot be undone.';
+  }
```

Comment:
nitpick: we could keep these as constants, but either works

```suggestion
      const resetRootPasswordCopy = isLegacy
    ? 'Resetting your root password will automatically generate a new password. You can view the updated password on your database cluster summary page. '
    : 'Reset your root password if someone should no longer have access to the root user or if you believe your password may have been compromised. This will automatically generate a new password that you’ll be able to see on your database cluster summary page.';
  const deleteClusterCopy = isLegacy
    ? 'Deleting a database cluster is permanent and cannot be undone.'
    : 'Permanently remove an unused database cluster.';
```

Reply:
Done